### DISCUSSION TITLE
Love it

### DISCUSSION THREAD

**Discussion by alita-moore** [simple-login/app#866] [Kind words ❤️]
I'm up to 150 aliases now. I have about 10 domains. I feel in control of my online presence! 

I'm excited to see where simple login goes next.

**Comment by ghost** [comment]
Paid for premium, added by personal domain and loving it so far. Thanks!

**Comment by crazy-matt** [comment]
Same thing here. Big big thanks to the SimpleLogin team for doing something which matters and solves a problem which no single institution cares about so far. ❤️

**Comment by hiccups1980** [comment]
Many, many thanks from me too. Since I have been using simplelogin I no longer receive any spam, can effectively deactivate newsletter emails and have much more control over which emails I receive and which I do not.
Furthermore, I no longer have to argue with support about deleting my private data. I simply deactivate the email address and that's it.
